lib/package.gi

```diff
@@ -1677,6 +1677,17 @@ InstallGlobalFunction( LoadPackage, function( arg )
         # Notify the documentation (for the available version).
         LoadPackageDocumentation( info );
 
+        # Notify extensions provided by the package.
+        if IsBound( info.Dependencies ) and
+           IsBound( info.Dependencies.Extensions ) then
+          for entry in info.Dependencies.Extensions do
```

Comment:
Extensions are not dependencies, so why `info.Dependencies.Extensions` instead of `info.Extensions`?